contracts/api-spec.yaml

```diff
@@ -550,6 +566,26 @@ components:
           type: string
           example: '1xw1'
 
+    createStreamLogRequest:
+      description: A request to create a new stream log
+      properties:
+        streamId:
+          type: integer
+          example: '123'
+        message:
+          type: string
+          example: 'A log message'
+        dateTime:
+          type: string
+          pattern: '^\d{4}-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])T(0[0-9]|1[0-9]|2[0-4])(\+00:00)$'
+          example: '2021-02-04T08:21:23+00:00'
+        decoderSerial:
+          type: string
+          example: '1xw1'
+        encoderSerial:
+          type: string
+          example: '2Xe2'
```

Comment:
Would it be possible to remove these fields from the create stream request? Since we have the stream ID we can determine the encoder and decoder serial ourselves in the backend, we don't need to make the devices responsible for providing the serial number of the other device.